tools/PI/DevHome.PI/BarWindow.xaml

```diff
@@ -0,0 +1,157 @@
+            <Grid.RowDefinitions>
+                <RowDefinition Height="Auto"/>
+                <RowDefinition Height="*" />
+                <RowDefinition Height="Auto" />
+            </Grid.RowDefinitions>
+
+            <GridView x:Name="MainCommandGridView" Margin="6,0,2,0" HorizontalAlignment="Left">
+                <GridView.Resources>
+                    <Style TargetType="TextBlock">
+                        <Setter Property="FontFamily" Value="Segoe Fluent Icons"/>
+                        <Setter Property="FontSize" Value="20"/>
+                    </Style>
+                    <Style TargetType="Button">
+                        <Setter Property="BorderThickness" Value="0"/>
+                        <Setter Property="Background" Value="Transparent"/>
+                    </Style>
+                    <Style TargetType="controls:ProcessSelectionButton">
+                        <Setter Property="BorderThickness" Value="0"/>
+                        <Setter Property="Background" Value="Transparent"/>
+                    </Style>
+                </GridView.Resources>
+                <GridView.ItemsPanel>
+                    <ItemsPanelTemplate>
+                        <ItemsWrapGrid Orientation="Horizontal"/>
+                    </ItemsPanelTemplate>
+                </GridView.ItemsPanel>
+
+                <StackPanel x:Name="AllControls" Orientation="{x:Bind BarOrientation, Mode=OneWay}" HorizontalAlignment="Center" Background="Transparent">
+                    <controls:ProcessSelectionButton
+                         x:Uid="ProcessChooserButton"
+                         x:Name="ProcessChooserButton"
+                         Click="ProcessChooserButton_Click" HorizontalAlignment="Center" Margin="0">
+                        <TextBlock Text="&#xecaa;" Margin="0"  />
+                    </controls:ProcessSelectionButton>
+                    
+                    <!-- Per App controls -->
+                    <controls:AppStatusBar x:Name="AppStatus" Visibility="Visible" Margin="0,5" HorizontalAlignment="Center"/>
+                
+                    <!-- Per System controls -->                   
+                    <Button
+                        x:Uid="SwitchLayoutButton"
+                        x:Name="SwitchLayoutButton"
+                        Click="SwitchLayoutButton_Click" HorizontalAlignment="Center">
+                        <TextBlock Text="&#xe8b4;"/>
+                    </Button>
+                    <Button
+                        x:Name="LargeContentButton"
+                        Click="LargeContentButton_Click" HorizontalAlignment="Center">
+                        <TextBlock Text="&#xe8a1;"/>
+                    </Button>
+
+                    <!-- TODO When we add a tool, we should add it to the ExternalToolsMenu. -->
+                    <Button
+                         x:Name="ExternalToolsButton"
+                         x:Uid="ExternalToolsButton" HorizontalAlignment="Center">

```

Comment:
Nit: if we start putting attributes on different lines, keep putting attributes on different lines in that element.